content/03.methods.md

```diff
@@ -151,13 +151,13 @@ We use pdfminer [@url:https://pdfminersix.readthedocs.io/en/latest/index.html] t
 The extracted text is then fed into our word2vec model to construct a document embedding representation.
 We pass this representation onto our journal and manuscript search to identify journals based on the ten closest neighbors of individual papers as well as journal centroids.
 We implemented this search using the scikit-learn implementation of k-d trees.
-To run cost effectively on Amazon Web Services, we sharded the k-d trees into four trees.
+To run it more cost effectively on cloud computing environment, we sharded the k-d trees into four trees.
 
-Accompanying these recommendations, we also provide a neural network derived visualization of our training set and the article's position within it.
+Accompanying these recommendations, we also provided a neural network derived visualization of our training set and the article's position within it.
 We used SAUCIE [@doi:10.1101/2020.03.04.975177], an autoencoder designed to cluster single cell RNA-seq data, to build a two-dimensional embedding space that could be applied to newly generated preprints without retraining, a limitation of other approaches that we explored for visualizing entities expected to lie on a nonlinear manifold.
 We trained this model on document embeddings of PMC articles that did not contain a matching preprint version.
 We used the following parameters to train the model: a hidden size of 2, a learning rate of 0.001, lambda_b of 0, lambda_c of 0.001, and lambda_d of 0.001 for 2000 iterations.
 When a user requests a new document, we can then project the document on the pretrained model to generate a visualization in two-dimensional space.
-We illustrate our recommendations as a short list and provide access to our network visualization at [https://greenelab.github.io/annorxiver-journal-recommender/](https://greenelab.github.io/annorxiver-journal-recommender/).
+We illustrate our recommendations as a short list and provide access to our network visualization at [https://greenelab.github.io/preprint-similarity-search/](https://greenelab.github.io/preprint-similarity-search/).

```

Comment:
New brandname.

Reply:
The old app name is also mentioned a couple times under [Software and Data Availability](https://greenelab.github.io/annorxiver_manuscript/#software-and-data-availability)